manage_projects.rb

```diff
@@ -463,6 +465,34 @@ def validate_credentials(project)
   puts
 end
 
+def add_budget(project)
+  valid = false
+  while !valid
+    print "Budget amount (c.u./month): "
+    amount = gets.chomp
+    valid = begin
+      !!Integer(amount, 10)
```

Comment:
Why does this value need to be converted to a boolean?

Reply:
I suppose it isn't entirely necessary, as if an integer is returned it will be truthy.